### ISSUE TITLE
swan-provider is ignoring `aria2_candidate_dirs` and `aria2_download_dir` parameters from config.toml

### ISSUE THREAD

**Issue by Shekelme** [filswan/go-swan-provider#213]
It looks that `swan-provider` is ignoring `aria2_candidate_dirs` and `aria2_download_dir` parameters from config.toml
At least I see aria2 is storing downloaded files in `/tmp` folder in spite that I have configured custom folders for both these parameters. And there are no relatively fresh files in `aria2_download_dir`.

I used `swan-provider`before the upgrade and the behavior was different.

```admfc@lenovo92:~$ swan-provider -V
NAME:
    swan-provider
VERSION:
    2.1.0
```

**Comment by Shekelme** [comment]
[report.txt](https://github.com/filswan/go-swan-provider/files/10859710/report.txt)
The result of `check-health.py` execution.

**Comment by Normalnoise** [comment]
@Shekelme As your report.txt shows, you should set your deal-ask:
```
Query-ask: 
  ERROR: Please set your miner's ask: 'lotus-miner storage-deals set-ask --price 0 --verified-price 0 --min-piece-size 1MiB --max-piece-size 32GB' 
```

**Comment by Normalnoise** [comment]
@Shekelme this bug has been fixed in the latest version [v2.2.0-rc1](https://github.com/filswan/go-swan-provider/releases/tag/v2.2.0-rc1)
